.github/workflows/release.yaml

```diff
@@ -6,25 +6,25 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+"
 
 jobs:
-  # release_creation:
-  #   name: Prepares Release on GitHub
-  #   runs-on: ubuntu-latest
-  #   outputs:
-  #     id: ${{ steps.main_step.outputs.id }}
-  #   steps:
-  #     - name: Create Release
-  #       id: main_step
-  #       uses: actions/create-release@latest
-  #       env:
-  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
-  #       with:
-  #         tag_name: ${{ github.ref }}
-  #         release_name: testkube CLI ${{ github.ref }}
-  #         body: |
-  #           Changes in this Release
-  #           - Releasing new version of the testkube CLI for Kubectl.
-  #         draft: false
-  #         prerelease: false
+#   release_creation:
+#     name: Prepares Release on GitHub
+#     runs-on: ubuntu-latest
+#     outputs:
+#       id: ${{ steps.main_step.outputs.id }}
+#     steps:
+#       - name: Create Release
+#         id: main_step
+#         uses: actions/create-release@latest
+#         env:
+#           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+#         with:
+#           tag_name: ${{ github.ref }}
+#           release_name: testkube CLI ${{ github.ref }}
+#           body: |
+#             Changes in this Release
+#             - Releasing new version of the testkube CLI for Kubectl.
+#           draft: false
+#           prerelease: false
```

Comment:
Is there a reason to keep all these commented out lines? I know it's not part of your code, was just wondering

Reply:
Frankly, I don't think we need it, since the pipeline is working and doesn't require any changes. I can remove it if no one else would mind.

Reply:
Just remove it, it'll still be there in the Git history